### ISSUE TITLE
Error when setting up Bukkit server

### ISSUE THREAD

**Issue by tlaundal** [SpaceCP/Bugs#99]
When setting up a bukkit server (working around #98 by `touch`ing the file), I get this error after passing the URL to my jar file:

```
info: bukkit: Loading as default vanilla Minecraft server
info: bukkit: Loading as Bukkit Minecraft server
Unhandled rejection TypeError: Cannot read property 'getMethod' of null
    at d:\Tobias\Programming\Daemon\lib\daemon\api\ServerManagerAPI.js:172:36
    at forOwn (d:\Tobias\Programming\Daemon\node_modules\lodash\dist\lodash.js:2106:15)
    at Function.forEach (d:\Tobias\Programming\Daemon\node_modules\lodash\dist\lodash.js:3303:9)
    at d:\Tobias\Programming\Daemon\lib\daemon\api\ServerManagerAPI.js:171:23
    at forOwn (d:\Tobias\Programming\Daemon\node_modules\lodash\dist\lodash.js:2106:15)
    at Function.forEach (d:\Tobias\Programming\Daemon\node_modules\lodash\dist\lodash.js:3303:9)
    at d:\Tobias\Programming\Daemon\lib\daemon\api\ServerManagerAPI.js:164:19
    at tryCatcher (d:\Tobias\Programming\Daemon\node_modules\bluebird\js\main\util.js:26:23)
    at Promise._settlePromiseFromHandler (d:\Tobias\Programming\Daemon\node_modules\bluebird\js\main\promise.js:507:31)
    at Promise._settlePromiseAt (d:\Tobias\Programming\Daemon\node_modules\bluebird\js\main\promise.js:581:18)
    at Promise._settlePromises (d:\Tobias\Programming\Daemon\node_modules\bluebird\js\main\promise.js:697:14)
    at Async._drainQueue (d:\Tobias\Programming\Daemon\node_modules\bluebird\js\main\async.js:123:16)
    at Async._drainQueues (d:\Tobias\Programming\Daemon\node_modules\bluebird\js\main\async.js:133:10)
    at Immediate.Async.drainQueues [as _onImmediate] (d:\Tobias\Programming\Daemon\node_modules\bluebird\js\main\async.js:15:14)
    at processImmediate [as _immediateCallback] (timers.js:383:17)
```

**Comment by JamyDev** [comment]
mhmkay

**Comment by tlaundal** [comment]
This was fixed with #98, it seems.

**Comment by JamyDev** [comment]
I'm going to verify the control flow before accepting this as fixed :P
